lib/miyuki/notifier.rb

```diff
@@ -19,7 +19,12 @@ def initialize(notifier = nil)
     end
 
     def notify(title, message)
-      @notifier.notify(title, message) if has_notifier?
+      begin
+        @notifier.notify(title, message) if has_notifier?
+      rescue NoMethodError
+        #libnotify is not installed, maybe.
+        #someone may not want to install gnome on its server :V
```

Comment:
I'd prefer not to touch them, I'll just do a `.patch` file to disable notifications when needed (manual operation, you know what you're doing). It's another file to maintain, but I don't think it's a problem, as they can be generated (and applied) with just a command.

Still, the problem remains: if we implement the test steps related to notifications ([this](https://github.com/RoxasShadow/Miyuki/blob/master/features/tracker.feature#L29) and [that](https://github.com/RoxasShadow/Miyuki/blob/master/features/load_config.feature#L18) ), they have to be executed somehow to make the tests pass.